lib/tachikoma/application.rb

```diff
@@ -221,6 +221,23 @@ def cocoapods
       end
     end
 
+    def yarn
+      Dir.chdir("#{Tachikoma.repos_path}/#{@build_for}") do
+        sh(*['git', 'config', 'user.name', @commiter_name])
+        sh(*['git', 'config', 'user.email', @commiter_email])
+        sh(*['git', 'checkout', '-b', "tachikoma/update-#{@readable_time}", @base_remote_branch])
+        sh(*%w(yarn install))
+        sh(*%w(yarn outdated))
```

Comment:
Thanks, review.
Yes. We have to execute `yarn outdated` before `yarn upgrade`.

see at (ci-yarn-upgrade: keep npm dependencies tool like tachikoma) :
https://github.com/taichi/ci-yarn-upgrade/blob/master/src/upgrade-request.js#L109
^ same process: install -> outdated -> upgrade